testserver/testserver.go

```diff
@@ -178,6 +179,14 @@ func SecureOpt() TestServerOpt {
 	}
 }
 
+// StoreInMemOpt is a TestServer option that can be passed to NewTestServer
+// to enable storing database in memory
+func StoreInMemOpt() TestServerOpt {
+	return func(args *testServerArgs) {
```

Comment:
actually, can we make it so storing in memory is the default behavior?

so maybe change this to be a `storeOnDisk` flag which defaults to false and can be made true with an option